shipit.yml

```
@@ -1,11 +1,11 @@
 ci:
   require:
-    - 'Test'
+    - node-tests (12.22.0)
```

Unsure if this should be similar to quilt format or just 
```suggestion
    - 'Test (12.22.0)'
```

---

It should be `Test (12.22.0)` in polaris to match [the name of the action that is ran](https://github.com/Shopify/polaris-react/blob/main/.github/workflows/ci.yml#L7) (which in quilt is `node-tests`)